src/api/Main/BuisnessLayer/Shrooms.Domain/Services/Wall/WallService.cs

```diff
@@ -768,6 +768,26 @@ private async Task<IEnumerable<WallDto>> GetUserFollowedWalls(UserAndOrganizatio
                     Logo = x.Wall.Logo
                 })
                 .ToListAsync();
+
+            if (!walls.Any(wall => wall.Type == WallType.Main))
+            {
+                var mainWall = await _wallUsersDbSet
+                    .Include(wall => wall.Wall)
+                    .Where(wall => wall.Wall.Type == WallType.Main && wall.Wall.OrganizationId == userOrg.OrganizationId)
+                    .Select(x => new WallDto
+                    {
+                        Id = x.Wall.Id,
```

Comment:
Looks as duplicated code a bit..
Can't be query above extended, so that Main wall would be always added? In that case it probably would be more sense to select from from _wallDbSet_ instead of _wallUsersDbSet_. In plan SQL it would look like something like this:
```
SELECT DISTINCT w.Name, w.Id, w.Description, w.Type, w.Logo, wm.UserId
FROM Walls w
JOIN WallMembers wm on wm.WallId = w.Id or w.Id = 1
WHERE wm.UserId = @UserId
```